app/models/comment.py

```diff
@@ -0,0 +1,44 @@
+from .db import db, environment, SCHEMA
+from sqlalchemy.sql import func
+
+
+class Comment(db.Model):
+    __tablename__ = 'comments'
+
+    if environment == "production":
+        __table_args__ = {'schema': SCHEMA}
+
+    id = db.Column(db.Integer, primary_key=True)
+    user_id = db.Column(db.Integer, nullable=False)
+    type_id = db.Column(db.Integer, nullable=False)
+    _type = db.Column('type', db.String(20), nullable=False)
+    comment = db.Column(db.String(255), nullable=False)
+    created_at = db.Column(db.DateTime, default=func.now())
+    updated_at = db.Column(db.DateTime, default=func.now())
+
+    def __init__(self, user_id, type_id, type, comment):
```

Comment:
Is there a reason for using the __init__ on some of these models? (goes for multiple model files)

Reply:
Redefining the _init_ to do the validation.